packages/components/src/internal/util/Date.ts

```diff
@@ -104,32 +104,42 @@ export function getTimeFormat(): string {
     return toMomentFormatString('HH:mm:ss');
 }
 
-export function parseDate(dateStr: string, dateFormat?: string): Date {
+export function parseDate(dateStr: string, dateFormat?: string, minDate?: Date): Date {
```

Comment:
I'd recommend adding a test set within `describe('parseDate')` that covers `minDate` functionality.

Reply:
Yup, done.